tests/unit/domain/consensus/test_vrf_algorithm.py

```
@@ -0,0 +1,149 @@
+from datetime import datetime
+
+
+def list_of_dicts_to_set(list_of_dicts: list[dict]) -> set:
+    return set(map(lambda x: tuple(x.items()), list_of_dicts))
+
+
+# Reimplementamos la función localmente para los tests para evitar importación circular
+def get_validators_for_transaction_test(
+    nodes: list[dict],
+    num_validators: int | None = None,
+    rng=np.random.default_rng(seed=int(datetime.now().timestamp())),
```

Ruff (0.14.1)

Do not perform function call `np.random.default_rng` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)